frontend/src/components/App/App.test.tsx

```diff
@@ -13,75 +13,75 @@ const mockedGetFeed = getFeed as jest.Mock<ReturnType<typeof getFeed>>;
 const mockedGetTags = getTags as jest.Mock<ReturnType<typeof getTags>>;
 const mockedGetUser = getUser as jest.Mock<ReturnType<typeof getUser>>;
 
-it('Should render home', async () => {
-  act(() => {
-    store.dispatch(initializeApp());
-  });
-  mockedGetArticles.mockResolvedValueOnce({
-    articles: [],
-    articlesCount: 0,
-  });
-  mockedGetTags.mockResolvedValueOnce({ tags: [] });
-  mockedGetUser.mockImplementationOnce(jest.fn());
-  localStorage.clear();
+describe('App', () => {
+    it('Should render home', async () => {
+        act(() => {
+            store.dispatch(initializeApp());
+        });
+        mockedGetArticles.mockResolvedValueOnce({
+            articles: [],
+            articlesCount: 0,
+        });
+        mockedGetTags.mockResolvedValueOnce({ tags: [] });
+        mockedGetUser.mockImplementationOnce(jest.fn());
+        localStorage.clear();
 
-  await act(async () => {
-    await render(<App />);
-  });
+        render(<App />);
```

Comment:
Also 
` 
        await act(async () => {
            render(<App />);
        });
`  ? There is also a "warning"/error in test output. The other deprecated warnings are ok, of course "The object notation for `createSlice.extraReducers` is deprecated, and will be removed in RTK 2.0."

Reply:
done